tests/deploy_test.go

```diff
@@ -13,6 +15,78 @@ import (
 
 var emptyQueryString = ""
 
+func Test_Scaling_Up(t *testing.T) {
+
+	functionRequest := requests.CreateFunctionRequest{
+		Image:      "functions/alpine:latest",
+		Service:    "test-scaling-up",
+		Network:    "func_functions",
+		EnvProcess: "env",
+	}
+
+	deployStatus, deployErr := deploy(t, functionRequest)
+	if deployErr != nil {
+		t.Errorf(deployErr.Error())
+		t.Fail()
+		return
+	}
+
+	if deployStatus != http.StatusOK && deployStatus != http.StatusAccepted {
+		t.Errorf("got %d, wanted %d or %d", deployStatus, http.StatusOK, http.StatusAccepted)
+	}
+
+	invoke(t, functionRequest.Service, emptyQueryString, http.StatusOK)
+
+	err := scaleUp(t, functionRequest.Service, "10")
```

Comment:
This should probably be `scale` or `setScale` since whether it's up or down is really dependent on the state of the deployment not the request being made.